src/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/Oid.cs

```diff
@@ -55,10 +57,13 @@ public static Oid FromFriendlyName(String friendlyName, OidGroup group)
             return new Oid(oidValue, friendlyName, group);
         }
 
-        public static Oid FromOidValue(String oidValue, OidGroup group)
+        public static Oid FromOidValue(String oidValue, OidGroup group, bool disableActiveDirectorySearch = false)
```

Comment:
This particular change (adding new parameter with a default value) changes the public API surface. We can't do that kind of change in a PR like this.

Please remove this change from the PR.